exonum-java-binding/common/src/test/java/com/exonum/binding/common/proofs/list/ListProofUtils.java

```diff
@@ -64,6 +64,13 @@ static ListProofNode generateRightLeaningProofTree(int depth, ListProofNode leaf
     return root;
   }
 
+  /**
+   * Creates ListProof absence node with given Merkle root.
+   */
+  static ListProofOfAbsence proofOfAbsence(HashCode merkleRoot) {
+    return new ListProofOfAbsence(merkleRoot.asBytes());
```

Comment:
As ListProofOfAbsence internally *creates* a HashCode, we might expose
     such constructor atop of the one accepting byte[], and delegate to it here and there.